Documentation/proposals/202305-scrapeclasses.md

```diff
@@ -0,0 +1,223 @@
+## Scrape Classes
+
+* **Owners:**
+  * `nicolastakashi`
+  * `eronwright`
+
+* **Related Tickets:**
+  * https://github.com/prometheus-operator/prometheus-operator/issues/4121
+  * https://github.com/prometheus-operator/prometheus-operator/issues/3922
+  * https://github.com/prometheus-operator/prometheus-operator/issues/5947
+  * https://github.com/prometheus-operator/prometheus-operator/issues/5948
+
+This proposal introduces the concept of *scrape classes*, enabling users to utilize scrape configuration data provided by the administrator, through scrape objects such as PodMonitor, Probe, ServiceMonitor and ScrapeConfig.
+
+## Why
+
+Sometimes Prometheus administrators needs to provide default configurations for scrape objects, such as the definition of TLS certificates, when running Prometheus to scrape pods in an Istio mesh with strict mTLS as described in [Istio documentation](https://istio.io/latest/docs/ops/integrations/prometheus/#tls-settings).
+
+Another motivation is to improve feature parity amongst the monitor resources. The `PodMonitor` and `Probe` resources aren't at parity with `ServiceMonitor` because the latter allows for unsafe TLS settings.
```

Comment:
This motivation sounds a bit weird, we can improve feature parity by allowing unsafe TLS on CRD that are still lacking 🤔

Reply:
I think it might be related.
https://github.com/prometheus-operator/prometheus-operator/issues/3922
It shouldn't exists even on Service Monitors from what I understood.
Maybe I might be wrong 😅